src/pages/docs/wasi-http.mdx

```diff
@@ -0,0 +1,136 @@
+# WASI HTTP Incoming handler
+
+If you have a wasm component that exports the wasi:http/incoming-handler\@0.2.x world, you can easily deploy your application to Golem.
+This tutorial will walk you through deploying such an application to Golem.
+
+Let's assume that we have an application written in any supported language that implements the following world:
+
+```wit
+package golem:example;
+
+world api {
+  import wasi:http/types@0.2.3;
+  export wasi:http/incoming-handler@0.2.3;
+}
+```
+
+and we want to deploy this under the GET /api route on our domain.
+
+The way we are going to do this using Golem is by defining an [api-definition](/docs/concepts/api-definitions) with the http handler binding type.
+
+## Building and uploading the component
+
+The first step is to build your application into a wasm component file. The way to do this depends on your language and toolchain, but
+using Rust and Cargo the command might look like this:
+
+```bash
+$ cargo component build --release
+
+Generating bindings for example (src/bindings.rs)
+ Compiling example v0.1.0 (/Users/vigoo/tmp/doc-temp/example)
```

Comment:
I wouldn't give /Users/vigoo etc in the doc, if possible.

Reply:
How did I get here? :D

Reply:
You are used in all the other examples already :D

Reply:
I'll change it to /Users/golem

Reply:
done https://github.com/golemcloud/docs/pull/115/commits/4571976ecd570ef67b119b4a411135317c8ab5d4